React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -54,6 +54,15 @@ - (void)reactSetFrame:(CGRect)frame
            horizontalScrollerHeight:verticalScrollerWidth];
     [[[scrollView bridge] uiManager] setLocalData:localData forView:self];
   }
+    
+  NSMutableArray *subViews = [[NSMutableArray alloc] init];
```

Comment:
we would know how to big this array should be and use inithwithcapacity?

Reply:
Agreed, using the size of `subviews` should suffice